app/Http/Requests/CreateReviewRequest.php

```diff
@@ -56,7 +55,7 @@ public function getRating() : float
      *
      * @return string
      */
-    public function getReview() : string
+    public function getReview()
```

Comment:
If this method will return null or string, return type can be set as ?string

Reply:
@TatsuUkraine fixed it.